scipy/stats/_multivariate.py

```diff
@@ -786,6 +786,39 @@ def entropy(self, mean=None, cov=1):
         dim, mean, cov_object = self._process_parameters(mean, cov)
         return 0.5 * (cov_object.rank * (_LOG_2PI + 1) + cov_object.log_pdet)
 
+    def fit(self, x):
+        """Fit the multivariate normal to data `x`.
+
+        Parameters
+        ----------
+        x : ndarray (m, n)
+            Data the distribution is fitted to. Must have two axes.
+            The first axis of length `m` represents the number of vectors
+            the distribution is fitted to. The second axis of length `n`
+            determines the dimensionality of the fitted distribution.
+
+        Returns
+        -------
+        mean : ndarray (n, )
+            Estimated mean vector
+        cov : ndarray (n, n)
+            Estimated covariance matrix
+
+        """
+        # input validation
+        x = np.asarray(x)
+        if x.ndim != 2:
+            raise ValueError("`x` must be two-dimensional of shape (m, n).")
```

Comment:
`m` and `n` are not defined in the error message (only in the documentation). Rather than defining them, let's just remove them, since they don't add any information.
```suggestion
            raise ValueError("`x` must be two-dimensional.")
```
Incidentally, I think this will also obviate the need to do anything special when matching the message. (e.g. no use of `re`, although there are likely other ways of doing that.)